cli/azd/resources/scaffold/templates/resources.bicept

```diff
@@ -704,7 +710,7 @@ module redisConn './modules/set-redis-conn.bicep' = {
 }
 {{- end}}
 
-{{- if .Services}}
+{{- if (or .Services .DbCosmosMongo .DbRedis)}}
```

Comment:
This actually ends up fixing #4574 and is an idea I had in mind as well. Any thoughts on this approach @weikanglim?

Reply:
@JeffreyCA Thanks for tagging me on this! At a first glance, I'd probably recommend not exporting the KeyVault secret if there are no services to bind to.

Reply:
That makes sense too. I created a separate PR to address this here: https://github.com/Azure/azure-dev/pull/4692

Reply:
I've merged the changes, so @saragluna you may need to revert this and pull the changes from `main`